cardano-node/app/cardano-cli.hs

```diff
@@ -1,208 +1,39 @@
-    , command' "print-genesis-hash"             "Compute hash of a genesis file." $
-      PrintGenesisHash
-      <$> parseFilePath    "genesis-json"             "Genesis JSON file to hash."
-    ])
-  <|> subparser
-  (mconcat
-    [ commandGroup "Keys"
-    , command' "keygen"                         "Generate a signing key." $
-      Keygen
-      <$> parseFilePath    "secret"                   "Non-existent file to write the secret key to."
-      <*> parseFlag        "no-password"              "Disable password protection."
-    , command' "to-verification"                "Extract a verification key in its base64 form." $
-      ToVerification
-      <$> parseFilePath    "secret"                   "Secret key file to extract from."
-      <*> parseFilePath    "to"                       "Non-existent file to write the base64-formatted verification key to."
-    , command' "signing-key-public"             "Pretty-print a signing key's verification key (not a secret)." $
-      PrettySigningKeyPublic
-      <$> parseFilePath    "secret"                   "File name of the secret key to pretty-print."
-    , command' "signing-key-address"            "Print address of a signing key." $
-      PrintSigningKeyAddress
-      <$> parseNetworkMagic
-      <*> parseFilePath    "secret"                   "Secret key, whose address is to be printed."
-    , command' "migrate-delegate-key-from"      "Migrate a delegate key from an older version." $
-      MigrateDelegateKeyFrom
-      <$> parseSystemVersion
-      <*> parseFilePath    "to"                       "Output secret key file."
-      <*> parseFilePath    "from"                     "Secret key file to migrate."
-    ])
-  <|> subparser
-  (mconcat
-    [ commandGroup "Delegation"
-    , command' "redelegate"                     "Redelegate genesis authority to a different verification key." $
-      Redelegate
-      <$> parseProtocolMagicId "protocol-magic"
-      <*> (EpochNumber <$>
-            parseIntegral   "since-epoch"              "First epoch of effective delegation.")
-      <*> parseFilePath    "secret"                   "The genesis key to redelegate from."
-      <*> parseFilePath    "delegate-key"             "The operation verification key to delegate to."
-      <*> parseFilePath    "certificate"              "Non-existent file to write the certificate to."
-    , command' "check-delegation"               "Verify that a given certificate constitutes a valid delegation relationship betwen keys." $
-      CheckDelegation
-      <$> parseProtocolMagicId "protocol-magic"
-      <*> parseFilePath    "certificate"              "The certificate embodying delegation to verify."
-      <*> parseFilePath    "issuer-key"               "The genesis key that supposedly delegates."
-      <*> parseFilePath    "delegate-key"             "The operation verification key supposedly delegated to."
-    ])
-
-parseTestnetBalanceOptions :: Parser Genesis.TestnetBalanceOptions
```

Comment:
See comment about `parseClientCommand`. These should stay here.

Reply:
Done.